app/src/main/java/com/yacgroup/yacguide/extensions/PackageManager.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2023 Chrisian Sommer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.yacgroup.yacguide.extensions
+
+import android.content.Intent
+import android.content.pm.PackageInfo
+import android.content.pm.PackageManager
+import android.content.pm.ResolveInfo
+import android.os.Build
+
+fun PackageManager.queryIntentActivitiesCompat(intent: Intent, flags: Int): List<ResolveInfo> =
```

Comment:
What is a "compat"?
Would it make sense to call it `queryCustomIntentActivities()` or `queryIntentActivities_patch()` to make clear that we've patched the actual package manager?
Same for the other function below.

Reply:
"compat" stands for compatibility. I think it is also used by Google for a set of libraries `AppCompat` which are compatible with older versions.